compose/config/config.py

```diff
@@ -213,6 +214,9 @@ def find(base_dir, filenames):
             os.getcwd(),
             [ConfigFile(None, yaml.safe_load(sys.stdin))])
 
+    if filename.startswith('./http'):
```

Comment:
I think `./http` is kind of hack. But @sergeyklay were you talking about supporting all the protocols? Or you're saying you prefer to support stdin and local file system only?